pom.xml

```diff
@@ -79,7 +79,7 @@
     <version.org.codehaus.groovy.maven>1.0</version.org.codehaus.groovy.maven>
     <version.org.drools>6.4.0.Final</version.org.drools>
     <version.org.freemarker>2.3.23</version.org.freemarker>
-    <version.org.hawkular.commons>0.9.0.Final</version.org.hawkular.commons>
+    <version.org.hawkular.commons>0.9.1.Final-SNAPSHOT</version.org.hawkular.commons>
```

Comment:
@jotak, can you change this to:   0.9.1.Final-SRC-revision-cca1a8e40526c5c577b25a39b94ea0a52c178fd6

That way anyone building master won't need to build commons locally.